openmp/runtime/src/z_Linux_asm.S

```diff
@@ -1850,35 +1860,35 @@ __kmp_invoke_microtask:
 	addi	a1, fp, __tid
 
 	beqz	t3, .L_kmp_3
-	ld	a2, 0(t4)
+	REG_L	a2, 0*SZREG(t4)
 
 	addi	t3, t3, -1
 	beqz	t3, .L_kmp_3
-	ld	a3, 8(t4)
+	REG_L	a3, 1*SZREG(t4)
 
 	addi	t3, t3, -1
 	beqz	t3, .L_kmp_3
-	ld	a4, 16(t4)
+	REG_L	a4, 2*SZREG(t4)
 
 	addi	t3, t3, -1
 	beqz	t3, .L_kmp_3
-	ld	a5, 24(t4)
+	REG_L	a5, 3*SZREG(t4)
 
 	addi	t3, t3, -1
 	beqz	t3, .L_kmp_3
-	ld	a6, 32(t4)
+	REG_L	a6, 4*SZREG(t4)
 
 	addi	t3, t3, -1
 	beqz	t3, .L_kmp_3
-	ld	a7, 40(t4)
+	REG_L	a7, 5*SZREG(t4)
 
 	// Prepare any additional argument passed through the stack
-	addi	t4, t4, 48
+	addi	t4, t4, 6*SZREG
 	mv	t1, sp
 	j .L_kmp_2
 .L_kmp_1:
-	ld	t2, 0(t4)
-	sd	t2, 0(t1)
+	REG_L	t2, 0*SZREG(t4)
```

Comment:
I would drop the `*SZREG` here. It makes sense above where we use various multiples of SZREG.